### PR TITLE
Improve issue templates

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

In the current issue templates to share environment, we cannot paste outputs at once because the python one-line commands do not have prefixt such as `- Optuna version`. So we need to copy & paste three times at least. 

## Description of the changes

- Add prefix in the commands.
- Fix a typo and indent.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/questions-help-support.yml [hunk 1]
```diff
@@ -39,10 +39,10 @@ body:
     id: logs
     attributes:
       label: Error messages, stack traces, or logs
-      description: Please copy and paste any relevant error messages, stack traces, or log output. 
+      description: Please copy and paste any relevant error messages, stack traces, or log output.
       render: shell
     validations:
-       required: true
```
Comment (nzw0301) [inline .github/ISSUE_TEMPLATE/questions-help-support.yml:45]: Unnecessary space

**Comment by HideakiImamura** [comment]
@contramundum53 Could you review this PR?

**Comment by contramundum53** [comment]
Thanks! 
One comment: I think it would be better to use `python3` instead of `python` so that more people can copy & paste the code.

**Review by HideakiImamura — APPROVED** [review]
LGTM except the comment by @contramundum53.

**Comment by nzw0301** [comment]
Thanks. I've updated the pull request by following comments.

**Review by contramundum53 — APPROVED** [review]

**Pull request merged**